### ISSUE TITLE
Implement a CI/CD

### ISSUE THREAD

**Issue by giorgiozoppi** [openMF/digital-bank-ui#113]
We need a continuous deploy enviroment, I can provide an azure account for staging and production.
I just need a volunteer that is able to give me some times.
@RadheTians  are you free?

**Comment by RadheTians** [comment]
Sure @giorgiozoppi

**Comment by zambogo7** [comment]
> We need a continuous deploy enviroment, I can provide an azure account for staging and production. I just need a volunteer that is able to give me some times. @RadheTians are you free?

That sounds like a great plan! I'm definitely interested in helping out with the continuous deploy environment. I can commit to dedicating some time to getting everything set up and running smoothly. Let's coordinate our schedules and get started!

**Comment by giorgiozoppi** [comment]
We are going to use my linode account. We've to set at first a private docker registry:
https://www.digitalocean.com/community/tutorials/how-to-set-up-a-private-docker-registry-on-ubuntu-20-04
After that we need to push the minimum fineract-cn artifacts. 
@abhi40308  which are the minimum microservices to use from fineract-cn to enable your frontend? 
Please drop me a private mail at giorgio at apache.org

**Comment by zambogo7** [comment]
Okay on it, but I'm using windows 10. Should I change to ubuntu?

**Comment by giorgiozoppi** [comment]
Yes, try to give us an automated process a script or a couple of VM. Than we export those VM in the cloud and we're done.

**Comment by zambogo7** [comment]
Will do, thanks!

**Comment by giorgiozoppi** [comment]
VMs are better. You can put in a well known location in a bucket S3  then we get and we're ok,
